hknweb/events/templates/events/all_rsvps.html

```diff
@@ -0,0 +1,124 @@
+        {% endfor %}
+    </div>
+
+    <div class="aligned">
+        <!-- Left side of page, not-rsvp'd events -->
+        <div class="left-half">
+
+            <h3>Not RSVP'd</h3>
+
+            {% if not_rsvpd_events %}
+                <table class="full-table">
+                    <thead>
+                        <th>Title</th>
+                        <th>Time</th>
+                        <th>Location</th>
+                        <th>Action</th>
+                    </thead>
+
+                    <tbody>
+                    {% for event in not_rsvpd_events %}
+                        <tr>
+                            <td align="center">
+                                <a href="{% url 'events:detail' event.id %}" class="rounded-text-box"
+                                   style="background-color: {{ event.event_type.color }};">
+                                    {{ event.name }}
+                                </a>
+                            </td>
+                            <td align="center">{{ event.start_time }}</td>
+                            <td align="center">{{ event.location }}</td>
+                            <td align="center">
+                                <form action="{% url 'events:rsvp' event.id %}" method="POST">
+                                    {% csrf_token %}
+                                    <input type="hidden" name="event" value="{{ event.id }}" />
+                                    <input type="hidden" name="next" value="{{ request.get_full_path }}">
+                                    <input type="submit" value="RSVP" />
+                                </form>
+                            </td>
+                        </tr>
+                    {% endfor %}
+                    </tbody>
+                </table>
+            {% else %}
+                <p>No events to show.</p>
+            {% endif %}
+        </div>
+
+        <!-- Right side of page, rsvp'd events -->
+        <div id='agenda' class="right-half">
+
+            <h3>RSVP'd / Waitlist</h3>
+
+            {% if rsvpd_events %}
+                <table class="full-table">
+                    <thead>
+                        <th>Title</th>
+                        <th>Time</th>
+                        <th>Location</th>
+                        <th>Action</th>
+                    </thead>
+
+                    <tbody>
+                    {% for event in rsvpd_events %}
+                        <tr>
+                            <td align="center">
+                                <a href="{% url 'events:detail' event.id %}" class="rounded-text-box"
+                                   style="background-color: {{ event.event_type.color }};">
+                                    {{ event.name }}
+                                </a>
+                                {% if event.waitlisted %}
+                                    <img src="{% static 'candidate/maybe.png'%}" alt="waitlisted" class="table-icon">
```

Comment:
It's not immediately clear that this picture represents that the user is currently waitlisted for the event. Perhaps a legend or another signifier?

Reply:
Is this fine? 
![Screen Shot 2020-07-19 at 6 21 21 PM](https://user-images.githubusercontent.com/28752320/87890897-23023200-c9ed-11ea-8ffd-bc31bf80c742.png)